src/ray/gcs/gcs_server/gcs_placement_group_manager.h

```diff
@@ -324,10 +332,9 @@ class GcsPlacementGroupManager : public rpc::PlacementGroupInfoHandler {
 
   /// The pending placement_groups which will not be scheduled until there's a resource
   /// change.
```

Comment:
Can you update the comment to describe the key semantics and how this reduces resource usage?